### ISSUE TITLE
Red magnifying glass appearing in the URL bar when a page takes a while to reload

### ISSUE THREAD

**Issue by duckaleuze** [datguypiko/Firefox-Mod-Blur#152]
![img-2024-09-21-18-52-38](https://github.com/user-attachments/assets/d637c0e7-200a-441a-b8b5-301f4e0b5794)

I'm not sure if this is on purpose, but if not, it should be white to better combine with the rest of the UI.

-selected mods: 
![image](https://github.com/user-attachments/assets/acc67fee-d1df-424b-ac62-77e2c67624b9)

btw, just wanted to thank you for this incredible theme, it's so good I'm glad to report issues for its improvement :D

**Comment by datguypiko** [comment]
Search icon should be hidden by default unless it's some very specific situation.

The reason you are having it I think is because you are visiting not secured or bad certificate website so it shows it red. 

Anyway I changed it so shouldnt be showed anymore.

Redownload userChrome.css 
https://github.com/datguypiko/Firefox-Mod-Blur/blob/master/userChrome.css

Anyway I changed it so shouldnt be showed anymore.

Redownload userChrome.css 
https://github.com/datguypiko/Firefox-Mod-Blur/blob/master/userChrome.css
